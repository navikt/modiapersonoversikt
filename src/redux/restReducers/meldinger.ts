import { getMeldinger } from '../../api/meldinger-api';
import { createActionsAndReducer } from './restResource';

const { reducer, action, reload, tilbakestill } = createActionsAndReducer('meldinger');

export function hentMeldinger(fødselsnummer: string) {
    return action(() => getMeldinger(fødselsnummer));
}

export function reloadMeldinger(fødselsnummer: string) {
    return reload(() => getMeldinger(fødselsnummer));
}

export function resetMeldingerResource() {
    return tilbakestill;
}

export default reducer;
