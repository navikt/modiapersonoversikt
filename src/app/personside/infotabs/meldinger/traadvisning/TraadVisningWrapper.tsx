import { AlertStripeInfo } from 'nav-frontend-alertstriper';
import Verktoylinje from './verktoylinje/Verktoylinje';
import TraadVisning from './TraadVisning';
import * as React from 'react';
import { Traad } from '../../../../../models/meldinger/meldinger';
import styled from 'styled-components/macro';
import usePrinter from '../../../../../utils/print/usePrinter';
import NyTraadVisning from './NyTraadVisning';
import IfFeatureToggleOn from '../../../../../components/featureToggle/IfFeatureToggleOn';
import { FeatureToggles } from '../../../../../components/featureToggle/toggleIDs';
import IfFeatureToggleOff from '../../../../../components/featureToggle/IfFeatureToggleOff';

interface TraadVisningWrapperProps {
    valgtTraad?: Traad;
    sokeord: string;
}

const StyledArticle = styled.article`
    display: flex;
    flex-direction: column;
`;

function TraadVisningWrapper(props: TraadVisningWrapperProps) {
    const printer = usePrinter();
    if (!props.valgtTraad) {
        return <AlertStripeInfo>Ingen melding valgt</AlertStripeInfo>;
    }
    return (
        <StyledArticle key={props.valgtTraad.traadId} role="tabpanel">
            <Verktoylinje valgtTraad={props.valgtTraad} visPrinter={true} />
            <IfFeatureToggleOn toggleID={FeatureToggles.useNewDialogComponents}>
                <NyTraadVisning sokeord={props.sokeord} valgtTraad={props.valgtTraad} printer={printer} />
            </IfFeatureToggleOn>
            <IfFeatureToggleOff toggleID={FeatureToggles.useNewDialogComponents}>
                <TraadVisning sokeord={props.sokeord} valgtTraad={props.valgtTraad} printer={printer} />
            </IfFeatureToggleOff>
        </StyledArticle>
    );
}

export default TraadVisningWrapper;
