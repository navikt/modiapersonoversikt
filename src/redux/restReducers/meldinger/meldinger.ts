import { createRestResourceReducerAndActions } from '../../../rest/utils/restResource';
import { apiBaseUri } from '../../../api/config';
import { AppState } from '../../reducers';
import { Traad } from '../../../models/meldinger/meldinger';

export function getMeldingerFetchUri(state: AppState): string {
    const fnr = state.gjeldendeBruker.f√∏dselsnummer;
    return `${apiBaseUri}/dialog/${fnr}/meldinger`;
}

export default createRestResourceReducerAndActions<Traad[]>('meldinger', getMeldingerFetchUri);
