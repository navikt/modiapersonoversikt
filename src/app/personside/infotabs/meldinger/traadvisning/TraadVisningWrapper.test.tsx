import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TraadVisningWrapper from './TraadVisningWrapper';
import TestProvider from '../../../../../test/Testprovider';
import { getTestStore, setupReactQueryMocks } from '../../../../../test/testStore';
import { statiskTraadMock } from '../../../../../mock/meldinger/statiskTraadMock';

test('Viser traad med verktÃ¸ylinje', () => {
    setupReactQueryMocks();
    const store = getTestStore();
    const traader = [statiskTraadMock];

    const container = renderer.create(
        <TestProvider customStore={store}>
            <TraadVisningWrapper valgtTraad={traader[0]} sokeord={''} />
        </TestProvider>
    );

    const json = container.toJSON();
    expect(json).toMatchSnapshot();
});
