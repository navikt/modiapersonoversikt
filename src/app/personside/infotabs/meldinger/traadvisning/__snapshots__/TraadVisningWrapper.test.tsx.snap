// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viser traad med verktøylinje 1`] = `
.c8 {
  position: relative;
  text-align: center;
  padding-top: 1rem;
}

.c9 > * {
  margin-top: 1rem;
}

.c19 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c5 {
  position: relative;
  border: none;
  padding: 0;
  border-radius: .35rem;
  cursor: pointer;
  background-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #0067c5;
}

.c5:after {
  border-bottom: 1px #b7b1a9 solid;
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%;
}

.c5:focus {
  outline: none;
  box-shadow: 0 0 0 0.1875rem #254b6d;
}

.c5:hover {
  opacity: 0.8;
}

.c5:hover:after {
  content: '';
  border-color: #0067c5;
}

.c11 {
  position: relative;
}

.c13 {
  position: absolute !important;
  bottom: 0.4rem;
  right: 0.4rem;
}

.c12 {
  position: relative;
}

.c12 .skjemaelement {
  margin-bottom: 0;
}

.c10 {
  padding-top: 1rem;
}

.c10 .inputPanelGruppe__inner {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c10 .inputPanelGruppe__inner > * {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c10 label {
  font-weight: 600;
  margin-bottom: 0.1rem;
}

.c10 .skjemaelement {
  margin-bottom: 0.7rem;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c14 button {
  margin: 0;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c6 {
  margin-right: 0.5rem;
}

.c1 {
  border-radius: 0.25rem;
  background-color: white;
  box-shadow: 0 0 0.5rem rgba(0,0,0,0.1),0 0 0 1px rgba(0,0,0,0.1);
  padding: .5rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 0.24rem;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c3 > *:not(:last-child) {
  margin-right: 1.5rem;
}

.c4 {
  color: #3e3832;
}

.c17 .snakkeboble-panel {
  -webkit-flex-basis: 40rem;
  -ms-flex-preferred-size: 40rem;
  flex-basis: 40rem;
}

.c17 a {
  word-break: break-word;
}

.c17 .snakkeboble {
  margin: 0;
}

.c20 {
  font-weight: 600;
}

.c18 {
  text-align: left;
}

.c18 em {
  font-style: normal;
  background-color: #eed28c;
  border-radius: 0.625rem;
  padding: 0 0.3125rem;
}

.c18 > *:not(:last-child) {
  border-bottom: solid 0.0625rem #b7b1a9;
  padding-bottom: 0.7rem;
  margin-bottom: 0.5rem;
}

.c22 {
  color: #0067c5;
}

.c21 .ekspanderbartPanel__hode,
.c21 .ekspanderbartPanel__innhold {
  padding: 0.3rem 0;
}

.c23 {
  display: none;
}

.c16 {
  padding: .5rem;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c16 > *:last-child {
  margin-top: .5rem;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c7 {
  position: relative;
  border: none;
  padding: 0;
  border-radius: .35rem;
  cursor: pointer;
  background-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #0067c5;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #3e3832;
}

.c7:after {
  border-bottom: 1px #b7b1a9 solid;
  position: absolute;
  bottom: 0;
  left: 0;
  width: 100%;
}

.c7:focus {
  outline: none;
  box-shadow: 0 0 0 0.1875rem #254b6d;
}

.c7:hover {
  opacity: 0.8;
}

.c7:hover:after {
  content: '';
  border-color: #0067c5;
}

@media print {
  .c17 {
    page-break-inside: avoid;
    border: solid 0.0625rem #78706a;
    margin-bottom: 2rem;
  }

  .c17 .snakkeboble__snakkeboble-pil-container,
  .c17 .nav-ikon,
  .c17 .bruker-ikon {
    display: none;
  }

  .c17 .snakkeboble-panel {
    -webkit-flex-basis: 100%;
    -ms-flex-preferred-size: 100%;
    flex-basis: 100%;
  }
}

@media print {
  .c21 {
    display: none;
  }
}

@media print {
  .c23 {
    display: inline-block;
  }
}

<article
  className="c0"
  role="tabpanel"
>
  <article
    aria-describedby="Helt tilfeldig ID"
    className="c1"
  >
    <h3
      className="sr-only"
      id="Helt tilfeldig ID"
      tabIndex={-1}
    >
      Verktøylinje - 
      Delsvar - Arbeid
    </h3>
    <div
      className="c2"
    >
      <div
        className="c3"
      >
        <button
          aria-expanded={false}
          className="c4 c5"
          onClick={[Function]}
        >
          <span
            className="c6"
          >
            <span
              className="typo-normal"
            >
              Journalfør
            </span>
          </span>
          <i
            className="nav-frontend-chevron chevronboks chevron--ned"
          />
        </button>
        <button
          aria-expanded={false}
          className="c4 c5"
          onClick={[Function]}
        >
          <span
            className="c6"
          >
            <span
              className="typo-normal"
            >
              Lag oppgave
            </span>
          </span>
          <i
            className="nav-frontend-chevron chevronboks chevron--ned"
          />
        </button>
        <button
          aria-expanded={false}
          className="c4 c5"
          onClick={[Function]}
        >
          <span
            className="c6"
          >
            <span
              className="typo-normal"
            >
              Merk
            </span>
          </span>
          <i
            className="nav-frontend-chevron chevronboks chevron--ned"
          />
        </button>
      </div>
      <a
        className="-a c7"
        download={true}
        href="/dialog/10108000398/sg838exr/print"
      >
        <p
          className="typo-normal"
        >
          Skriv ut
        </p>
      </a>
    </div>
    <section
      className="c8"
    >
      <div
        className="c9"
      >
        <div
          className="alertstripe alertstripe--info"
        >
          <span
            aria-label="info"
            className="alertstripe__ikon"
          >
            <svg
              focusable="false"
              height="1.5em"
              kind="info-sirkel-fyll"
              viewBox="0 0 24 24"
              width="1.5em"
            >
              <g
                fill="none"
              >
                <path
                  d="M12 0C5.382 0 0 5.382 0 12s5.382 12 12 12c6.617 0 12-5.382 12-12S18.617 0 12 0z"
                  fill="#669DB4"
                />
                <path
                  d="M12 5a1.566 1.566 0 1 1 .11 3.13A1.566 1.566 0 0 1 12 5zm2.976 12.01c.563 0 1.043.431 1.043.991s-.48.992-1.043.992H9.39c-.564 0-1.043-.431-1.043-.992 0-.56.479-.99 1.043-.99h1.6v-5.016h-.986c-.565 0-1.044-.43-1.044-.991 0-.56.48-.991 1.044-.991h2.03c.563 0 1.043.43 1.043.99v6.007h1.899z"
                  fill="#FFF"
                />
              </g>
            </svg>
          </span>
          <div
            className="typo-normal alertstripe__tekst"
          >
            Tråden er allerede journalført
          </div>
        </div>
        <button
          className="knapp knapp--hoved"
          disabled={false}
          onClick={[Function]}
          type="submit"
        >
          Lukk
        </button>
      </div>
    </section>
    <div
      className="c10"
    >
      <form
        onSubmit={[Function]}
      >
        <div
          className="skjemaelement"
        >
          <label
            className="skjemaelement__label"
            htmlFor="Helt tilfeldig ID"
          >
            Tema
          </label>
          <div
            className="selectContainer input--fullbredde"
          >
            <select
              className="skjemaelement__input"
              disabled={false}
              id="Helt tilfeldig ID"
              onChange={[Function]}
              value=""
            >
              <option
                disabled={true}
                value=""
              >
                Velg tema
              </option>
              <option
                value="AGR"
              >
                Ajourhold - Grunnopplysninger
              </option>
              <option
                value="AAP"
              >
                Arbeidsavklaringspenger
              </option>
              <option
                value="BID"
              >
                Bidrag
              </option>
            </select>
          </div>
          <div
            aria-live="assertive"
            role="alert"
          />
        </div>
        <div
          className="skjemaelement"
        >
          <label
            className="skjemaelement__label"
            htmlFor="Helt tilfeldig ID"
          >
            Gjelder
          </label>
          <div
            className="selectContainer input--fullbredde"
          >
            <select
              className="skjemaelement__input"
              disabled={false}
              id="Helt tilfeldig ID"
              onChange={[Function]}
              value=""
            >
              <option
                disabled={true}
                value=""
              >
                Ingen tema valgt
              </option>
            </select>
          </div>
          <div
            aria-live="assertive"
            role="alert"
          />
        </div>
        <div
          className="skjemaelement"
        >
          <label
            className="skjemaelement__label"
            htmlFor="Helt tilfeldig ID"
          >
            Type oppgave
          </label>
          <div
            className="selectContainer input--fullbredde"
          >
            <select
              className="skjemaelement__input"
              disabled={false}
              id="Helt tilfeldig ID"
              onChange={[Function]}
              value=""
            >
              <option
                disabled={true}
                value=""
              >
                Ingen tema valgt
              </option>
            </select>
          </div>
          <div
            aria-live="assertive"
            role="alert"
          />
        </div>
        <div
          aria-expanded={false}
          aria-haspopup="listbox"
          aria-labelledby="downshift-0-label"
          aria-owns={null}
          className="c11"
          role="combobox"
        >
          <div
            className="c12"
          >
            <div
              className="skjemaelement"
            >
              <label
                className="skjemaelement__label"
                htmlFor="downshift-0-input"
              >
                Velg enhet
              </label>
              <input
                aria-activedescendant={null}
                aria-autocomplete="list"
                aria-controls={null}
                aria-label="Laster data"
                aria-labelledby="downshift-0-label"
                autoComplete="off"
                className="skjemaelement__input input--fullbredde"
                id="downshift-0-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                type="text"
                value=""
              />
              <div
                aria-live="assertive"
                role="alert"
              />
            </div>
            <svg
              aria-label="Laster innhold"
              className="spinner c13 spinner--s"
              focusable="false"
              height={32}
              kind="spinner"
              preserveAspectRatio="xMidYMid"
              viewBox="0 0 50 50"
              width={32}
            >
              <title>
                Venter...
              </title>
              <circle
                cx="25"
                cy="25"
                fill="none"
                r="20"
                stroke="#bdbab7"
                strokeWidth="5"
                xmlns="http://www.w3.org/2000/svg"
              />
              <circle
                cx="25"
                cy="25"
                fill="none"
                r="20"
                stroke="#7f756c"
                strokeDasharray="50 155"
                strokeLinecap="round"
                strokeWidth="5"
              />
            </svg>
          </div>
        </div>
        <div
          aria-expanded={false}
          aria-haspopup="listbox"
          aria-labelledby="downshift-1-label"
          aria-owns={null}
          className="c11"
          role="combobox"
        >
          <div
            className="c12"
          >
            <div
              className="skjemaelement"
            >
              <label
                className="skjemaelement__label"
                htmlFor="downshift-1-input"
              >
                Velg ansatt
              </label>
              <input
                aria-activedescendant={null}
                aria-autocomplete="list"
                aria-controls={null}
                aria-label="Laster data"
                aria-labelledby="downshift-1-label"
                autoComplete="off"
                className="skjemaelement__input input--fullbredde"
                id="downshift-1-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                type="text"
                value=""
              />
              <div
                aria-live="assertive"
                role="alert"
              />
            </div>
            <svg
              aria-label="Laster innhold"
              className="spinner c13 spinner--s"
              focusable="false"
              height={32}
              kind="spinner"
              preserveAspectRatio="xMidYMid"
              viewBox="0 0 50 50"
              width={32}
            >
              <title>
                Venter...
              </title>
              <circle
                cx="25"
                cy="25"
                fill="none"
                r="20"
                stroke="#bdbab7"
                strokeWidth="5"
                xmlns="http://www.w3.org/2000/svg"
              />
              <circle
                cx="25"
                cy="25"
                fill="none"
                r="20"
                stroke="#7f756c"
                strokeDasharray="50 155"
                strokeLinecap="round"
                strokeWidth="5"
              />
            </svg>
          </div>
        </div>
        <div
          className="skjemaelement"
        >
          <label
            className="skjemaelement__label"
            htmlFor="Helt tilfeldig ID"
          >
            Velg prioritert
          </label>
          <div
            className="selectContainer input--fullbredde"
          >
            <select
              className="skjemaelement__input"
              disabled={false}
              id="Helt tilfeldig ID"
              onChange={[Function]}
              value=""
            >
              <option
                disabled={true}
                value=""
              >
                Ingen tema valgt
              </option>
            </select>
          </div>
          <div
            aria-live="assertive"
            role="alert"
          />
        </div>
        <div
          className="skjemaelement textarea__container"
        >
          <label
            className="skjemaelement__label"
            htmlFor="Helt tilfeldig ID"
          >
            Beskrivelse
          </label>
          <div
            className="textarea--medMeta__wrapper"
          >
            <textarea
              className="skjemaelement__input textarea--medMeta"
              id="Helt tilfeldig ID"
              onChange={[Function]}
              style={
                Object {
                  "height": "30px",
                }
              }
              type="text"
              value=""
            />
          </div>
          <div
            aria-live="assertive"
            role="alert"
          />
          <div
            aria-hidden="true"
            className="textareamirror"
          />
        </div>
        <div
          className="c14"
        >
          <button
            className="knapp knapp--hoved"
            disabled={false}
            type="submit"
          >
            Opprett oppgave
          </button>
          <button
            className="c5"
            onClick={[Function]}
            type="button"
          >
            Avbryt
          </button>
        </div>
      </form>
    </div>
    <form
      onSubmit={[Function]}
    >
      <div
        className="inputPanelGruppe"
      >
        <fieldset
          className="skjema__fieldset"
        >
          <legend
            className="skjema__legend"
          >
            
          </legend>
          <div
            className="inputPanelGruppe__inner"
          >
            <label
              className="inputPanel radioPanel inputPanel--disabled"
              htmlFor="Helt tilfeldig ID"
            >
              <input
                aria-checked={false}
                checked={false}
                className="inputPanel__field"
                disabled={true}
                id="Helt tilfeldig ID"
                name="merk"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                type="radio"
                value="FEILSENDT"
              />
              <span
                className="inputPanel__label"
              >
                Feilsendt post
              </span>
            </label>
            <label
              className="inputPanel radioPanel inputPanel--disabled"
              htmlFor="Helt tilfeldig ID"
            >
              <input
                aria-checked={false}
                checked={false}
                className="inputPanel__field"
                disabled={true}
                id="Helt tilfeldig ID"
                name="merk"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                type="radio"
                value="BISYS"
              />
              <span
                className="inputPanel__label"
              >
                Kopiert inn i Bisys
              </span>
            </label>
            <label
              className="inputPanel radioPanel inputPanel--disabled"
              htmlFor="Helt tilfeldig ID"
            >
              <input
                aria-checked={false}
                checked={false}
                className="inputPanel__field"
                disabled={true}
                id="Helt tilfeldig ID"
                name="merk"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                type="radio"
                value="KONTORSPERRET"
              />
              <span
                className="inputPanel__label"
              >
                Kontorsperret
              </span>
            </label>
            <label
              className="inputPanel radioPanel inputPanel--disabled"
              htmlFor="Helt tilfeldig ID"
            >
              <input
                aria-checked={false}
                checked={false}
                className="inputPanel__field"
                disabled={true}
                id="Helt tilfeldig ID"
                name="merk"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                type="radio"
                value="AVSLUTT"
              />
              <span
                className="inputPanel__label"
              >
                Avslutt uten å svare bruker
              </span>
            </label>
          </div>
        </fieldset>
        <div
          aria-live="assertive"
          role="alert"
        />
      </div>
      <div
        className="c15"
      >
        <button
          className="knapp knapp--hoved"
          disabled={false}
          type="submit"
        >
          Merk
        </button>
        <button
          className="c5"
          onClick={[Function]}
          type="button"
        >
          Lukk
        </button>
      </div>
    </form>
  </article>
  <section
    className="c16"
  >
    <div
      className="alertstripe alertstripe--info"
    >
      <span
        aria-label="info"
        className="alertstripe__ikon"
      >
        <svg
          focusable="false"
          height="1.5em"
          kind="info-sirkel-fyll"
          viewBox="0 0 24 24"
          width="1.5em"
        >
          <g
            fill="none"
          >
            <path
              d="M12 0C5.382 0 0 5.382 0 12s5.382 12 12 12c6.617 0 12-5.382 12-12S18.617 0 12 0z"
              fill="#669DB4"
            />
            <path
              d="M12 5a1.566 1.566 0 1 1 .11 3.13A1.566 1.566 0 0 1 12 5zm2.976 12.01c.563 0 1.043.431 1.043.991s-.48.992-1.043.992H9.39c-.564 0-1.043-.431-1.043-.992 0-.56.479-.99 1.043-.99h1.6v-5.016h-.986c-.565 0-1.044-.43-1.044-.991 0-.56.48-.991 1.044-.991h2.03c.563 0 1.043.43 1.043.99v6.007h1.899z"
              fill="#FFF"
            />
          </g>
        </svg>
      </span>
      <div
        className="typo-normal alertstripe__tekst"
      >
        Henvendelsen er avsluttet uten å svare bruker av 
        Ukjent saksbehandler
         
      </div>
    </div>
    <h3
      aria-live="polite"
      className="sr-only"
    >
      Delsvar - Arbeid
       
      13.08.2019 00:00
    </h3>
    <ol
      aria-label="Dialog"
    >
      <li
        className="snakkeboble_ikoner c17"
      >
        <article
          aria-describedby="Helt tilfeldig ID"
        >
          <div
            className="snakkeboble snakkeboble--hoyre"
          >
            <i
              className="nav-ikon"
            />
            <div
              className="snakkeboble__snakkeboble-pil-container"
            >
              <i
                className="snakkeboble__snakkebole-pil"
              />
            </div>
            <div
              className="panel snakkeboble-panel"
            >
              <div
                className="snakkeboble-panel__tekst"
              >
                <div
                  className="c18"
                >
                  <div
                    className=""
                  >
                    <div
                      className="c19"
                    >
                      <h4
                        id="Helt tilfeldig ID"
                      >
                        <div
                          className="tekstomrade c20"
                        >
                          Delsvar - Arbeid
                        </div>
                      </h4>
                      <div
                        className="etikett etikett--info"
                      >
                        <span
                          className="typo-normal"
                        >
                          Delsvar
                        </span>
                      </div>
                    </div>
                    <div
                      className="tekstomrade"
                    >
                      27.07.2019 00:00
                    </div>
                    <div
                      className="tekstomrade typo-normal"
                    >
                      Skrevet av Kristoffer Fredriksen (xy70r8)
                    </div>
                  </div>
                  <div
                    className="tekstomrade"
                  >
                    Dolores omnis ea voluptatem corrupti odit incidunt enim. Unde fugiat sed animi explicabo maiores. Nihil cum accusamus omnis officiis deserunt est. Qui quisquam unde aut magni dignissimos.
                  </div>
                  <div
                    className="ekspanderbartPanel c21 ekspanderbartPanel--lukket"
                  >
                    <button
                      aria-expanded={false}
                      className="ekspanderbartPanel__hode"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      type="button"
                    >
                      <div
                        className="ekspanderbartPanel__flex-wrapper"
                      >
                        <p
                          className="typo-etikett-liten c22"
                        >
                          Meldingen er journalført
                        </p>
                        <span
                          className="ekspanderbartPanel__indikator"
                        />
                      </div>
                    </button>
                    <article
                      className="ekspanderbartPanel__innhold"
                    >
                      Journalført av Jenny Berg (p0jh0d) 30.07.2019 på ukjent tema med ukjent saksid
                    </article>
                  </div>
                  <div
                    className="c23"
                  >
                    <p
                      className="typo-element"
                    >
                      Journalført av Jenny Berg (p0jh0d) 30.07.2019 på ukjent tema med ukjent saksid
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </li>
      <li
        className="snakkeboble_ikoner c17"
      >
        <article
          aria-describedby="Helt tilfeldig ID"
        >
          <div
            className="snakkeboble snakkeboble--venstre"
          >
            <i
              className="bruker-ikon"
            />
            <div
              className="snakkeboble__snakkeboble-pil-container"
            >
              <i
                className="snakkeboble__snakkebole-pil"
              />
            </div>
            <div
              className="panel snakkeboble-panel"
            >
              <div
                className="snakkeboble-panel__tekst"
              >
                <div
                  className="c18"
                >
                  <div
                    className=""
                  >
                    <div
                      className="c19"
                    >
                      <h4
                        id="Helt tilfeldig ID"
                      >
                        <div
                          className="tekstomrade c20"
                        >
                          Spørsmål fra bruker - Arbeid
                        </div>
                      </h4>
                    </div>
                    <div
                      className="tekstomrade"
                    >
                      16.07.2019 00:00
                    </div>
                  </div>
                  <div
                    className="tekstomrade"
                  >
                    Quo magni eaque et libero minima ipsum amet et dolorem. Sed iusto ratione sit. Voluptate alias qui impedit ut dolorum doloribus qui. Qui beatae rerum minima suscipit.
                  </div>
                  <div
                    className="ekspanderbartPanel c21 ekspanderbartPanel--lukket"
                  >
                    <button
                      aria-expanded={false}
                      className="ekspanderbartPanel__hode"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      type="button"
                    >
                      <div
                        className="ekspanderbartPanel__flex-wrapper"
                      >
                        <p
                          className="typo-etikett-liten c22"
                        >
                          Meldingen er journalført
                        </p>
                        <span
                          className="ekspanderbartPanel__indikator"
                        />
                      </div>
                    </button>
                    <article
                      className="ekspanderbartPanel__innhold"
                    >
                      Journalført av Emil Nguyen (ly4pps) 21.07.2019 på ukjent tema med ukjent saksid
                    </article>
                  </div>
                  <div
                    className="c23"
                  >
                    <p
                      className="typo-element"
                    >
                      Journalført av Emil Nguyen (ly4pps) 21.07.2019 på ukjent tema med ukjent saksid
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </li>
      <li
        className="snakkeboble_ikoner c17"
      >
        <article
          aria-describedby="Helt tilfeldig ID"
        >
          <div
            className="snakkeboble snakkeboble--venstre"
          >
            <i
              className="bruker-ikon"
            />
            <div
              className="snakkeboble__snakkeboble-pil-container"
            >
              <i
                className="snakkeboble__snakkebole-pil"
              />
            </div>
            <div
              className="panel snakkeboble-panel"
            >
              <div
                className="snakkeboble-panel__tekst"
              >
                <div
                  className="c18"
                >
                  <div
                    className=""
                  >
                    <div
                      className="c19"
                    >
                      <h4
                        id="Helt tilfeldig ID"
                      >
                        <div
                          className="tekstomrade c20"
                        >
                          Spørsmål fra bruker - Arbeid
                        </div>
                      </h4>
                    </div>
                    <div
                      className="tekstomrade"
                    >
                      12.07.2019 00:00
                    </div>
                  </div>
                  <div
                    className="tekstomrade"
                  >
                    Animi qui molestiae omnis reiciendis quam in. Provident magnam dolor. Omnis voluptatem placeat labore in ut aliquam perspiciatis fugiat. Perspiciatis voluptatum odit.
                  </div>
                  <div
                    className="ekspanderbartPanel c21 ekspanderbartPanel--lukket"
                  >
                    <button
                      aria-expanded={false}
                      className="ekspanderbartPanel__hode"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      type="button"
                    >
                      <div
                        className="ekspanderbartPanel__flex-wrapper"
                      >
                        <p
                          className="typo-etikett-liten c22"
                        >
                          Meldingen er journalført
                        </p>
                        <span
                          className="ekspanderbartPanel__indikator"
                        />
                      </div>
                    </button>
                    <article
                      className="ekspanderbartPanel__innhold"
                    >
                      Journalført av Victoria Andresen (3t55yt) 12.08.2019 på ukjent tema med ukjent saksid
                    </article>
                  </div>
                  <div
                    className="c23"
                  >
                    <p
                      className="typo-element"
                    >
                      Journalført av Victoria Andresen (3t55yt) 12.08.2019 på ukjent tema med ukjent saksid
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </li>
      <li
        className="snakkeboble_ikoner c17"
      >
        <article
          aria-describedby="Helt tilfeldig ID"
        >
          <div
            className="snakkeboble snakkeboble--hoyre"
          >
            <i
              className="nav-ikon"
            />
            <div
              className="snakkeboble__snakkeboble-pil-container"
            >
              <i
                className="snakkeboble__snakkebole-pil"
              />
            </div>
            <div
              className="panel snakkeboble-panel"
            >
              <div
                className="snakkeboble-panel__tekst"
              >
                <div
                  className="c18"
                >
                  <div
                    className=""
                  >
                    <div
                      className="c19"
                    >
                      <h4
                        id="Helt tilfeldig ID"
                      >
                        <div
                          className="tekstomrade c20"
                        >
                          Samtalereferat oppmøte - Arbeid
                        </div>
                      </h4>
                      <div
                        className="etikett etikett--suksess"
                      >
                        <span
                          className="typo-normal"
                        >
                          Lest
                        </span>
                      </div>
                    </div>
                    <div
                      className="tekstomrade"
                    >
                      07.07.2019 00:00
                    </div>
                    <div
                      className="tekstomrade typo-normal"
                    >
                      Skrevet av Sander Hansen (nzxo4q)
                    </div>
                  </div>
                  <div
                    className="tekstomrade"
                  >
                    Ea laborum accusamus et ex voluptates provident eveniet voluptatem. Sunt et sed eligendi harum exercitationem. Eaque est maiores autem sunt occaecati atque ipsa ea itaque. Dolorum enim quaerat alias ut sit labore pariatur.
                  </div>
                  <div
                    className="ekspanderbartPanel c21 ekspanderbartPanel--lukket"
                  >
                    <button
                      aria-expanded={false}
                      className="ekspanderbartPanel__hode"
                      onClick={[Function]}
                      onKeyDown={[Function]}
                      type="button"
                    >
                      <div
                        className="ekspanderbartPanel__flex-wrapper"
                      >
                        <p
                          className="typo-etikett-liten c22"
                        >
                          Meldingen er journalført
                        </p>
                        <span
                          className="ekspanderbartPanel__indikator"
                        />
                      </div>
                    </button>
                    <article
                      className="ekspanderbartPanel__innhold"
                    >
                      Journalført av Adrian Pettersen (vlzaiw) 11.07.2019 på ukjent tema med ukjent saksid
                    </article>
                  </div>
                  <div
                    className="c23"
                  >
                    <p
                      className="typo-element"
                    >
                      Journalført av Adrian Pettersen (vlzaiw) 11.07.2019 på ukjent tema med ukjent saksid
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </li>
    </ol>
  </section>
</article>
`;
